core/Columns/Dimension.php

```diff
@@ -456,7 +456,7 @@ public function formatValue($value, $idSite, Formatter $formatter)
             case Dimension::TYPE_DURATION_S:
                 return $formatter->getPrettyTimeFromSeconds($value, $displayAsSentence = false);
             case Dimension::TYPE_DURATION_MS:
-                $val = number_format($value / 1000, 2);
+                $val = number_format($value / 1000, 2, '.', '');
                 if ($val > 60) {
                     $val = round($val);
                 }
```

Comment:
The number format is actually only used to limit the precision. So can't that be simplified with
```suggestion
                $val = round($val / 1000, $val > 60 ? 0 : 2);
```

Reply:
makes sense and works a it is :+1:

Reply:
Updated as suggested